node/src/BaseClient.ts

```diff
@@ -209,6 +209,29 @@ export type ReturnType =
     | ReturnTypeAttribute
     | ReturnType[];
 
+export type GlideString = string | Uint8Array;
+
+export const enum Decoder {
+    Bytes,
+    String,
+}
+
+class PointerResponse {
+    pointer: number | Long | null;
+    high: number | undefined;
+    low: number | undefined;
```

Comment:
Why do you need the pointer and high + low altogether? Having a pointer you can calculate high & low bits at any time and vice versa.

Reply:
Why? you might know the high, but how do you know the current low? you read in iterations